modules/args-test/install.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
```

Comment:
This was my test case to validate the we are properly maintaining whitespace in args parameters.